### DISCUSSION TITLE
CCPP SCM 6.0.0: libzstd.so.1: cannot open error

### DISCUSSION THREAD

**Discussion by AnningCheng-NOAA** [NCAR/ccpp-scm#388] [Q&A]
I  have got an err by running the default case in CCPP SCM6.0.0: 
libzstd.so.1: cannot open shared object file: No such file or directory

I think I Have loaded needed modules, source scm/etc/Hera_setup_intel.csh
/scratch1/NCEPDEV/global/Anning.Cheng/ccpp-scm/scm/bin% module list
Currently Loaded Modules:
  1) hpc/1.1.0        3) hpc-intel/2022.1.2   5) hpc-impi/2022.1.2   7) hdf5/1.12.2    9) cmake/3.20.1  11) sp/2.3.3
  2) intel/2022.1.2   4) impi/2022.1.2        6) szip/2.1.1          8) netcdf/4.9.0  10) bacio/2.4.1   12) w3emc/2.9.2

**Comment by grantfirl** [comment]
@mkavulich @hertneky DTC folks, I can reproduce this issue on Hera too. Does anyone have time to see what is going on with the Hera software stack?

**Reply by mkavulich** [reply]
I will take a look

**Reply by mkavulich** [reply]
@grantfirl It looks like Tracy isn't a collaborator on this repo yet so I'm not sure if tagging her will work. I just added her with write permissions.

I am concerned that the fix I just posted is just a bandage fix and I don't 100% understand why this issue is occurring. The module files we are using haven't been updated since November, but I have only done testing on Cheyenne since then so the Hera capability may have been broken for months in that case. I wanted to test with develop but the online tutorial is out of date (uses the multirun script which no longer exists). Are there more up-to-date instructions somewhere?

**Reply by hertneky** [reply]
@mkavulich Thanks for the add Mike. I have also run into this issue in the past (a few months ago). I loaded the zstd library manually as well when this happened and it fixed it.

**Comment by mkavulich** [comment]
@AnningCheng-NOAA As a temporary fix, loading the zstd module (after setting up the rest of your environment with the setup script) should fix this issue.

`module load zstd/1.5.0`

**Comment by climbfuji** [comment]
Any interest in moving to spack-stack? The UFS will be moving over very soon.
